esp-box-lite/include/bsp/esp-box-lite.h

```diff
@@ -377,6 +361,29 @@ esp_err_t bsp_display_backlight_off(void);
  */
 void bsp_display_rotate(lv_disp_t *disp, lv_disp_rot_t rotation);
 
+/**************************************************************************************************
+ *
+ * ADC interface
+ *
+ * There are multiple devices connected to ADC peripheral:
+ *  - Buttons
+ *
+ * After initialization of ADC, use adc_handle when using ADC driver.
+ **************************************************************************************************/
+
+#define BSP_ADC_UNIT     ADC_UNIT_1
+
+/**
+ * @brief Initialize ADC
+ *
+ * The ADC can be initialized inside BSP, when needed.
+ * You can call this function more times and you get only the handle when initialized before.
+ * @warning When IDF version <5 returned adc_handle is NULL.
+ *
+ * @param[out] adc_handle Returned ADC handle
+ */
+esp_err_t bsp_adc_initialize(void **adc_handle);
```

Comment:
Here it's not clear what should be the type of `adc_handle`.

Perhaps we could split it into
- bsp_adc_initialize
- bsp_adc_get_handle — on for IDF >=v5.0

Reply:
I understand, you are right. I changed it. Thank you for review.